tmc-prototype/CentralNode/.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/CentralNode.iml" filepath="$PROJECT_DIR$/.idea/CentralNode.iml" />
```

Comment:
Ensure character limit to 110.

Reply:
This file is a part of PyCharm IDE. It will be ignored before merging to the master. However, character limit to 110 will be taken care for other code files.